mkt/langpacks/models.py

```diff
@@ -121,7 +121,7 @@ def handle_file_operations(self, upload):
         upload.path = smart_path(nfd_str(upload.path))
         if not self.uuid:
             self.reset_uuid()
-        if storage.exists(self.filename):
+        if private_storage.exists(self.filename):
```

Comment:
I think this was wrong to begin with and should be `self.file_path`. Furthermore, it should be `public_storage`, since this is the final, signed version we are looking at here.